### PR TITLE
[DOCS-6392] Set mutual TLS solr.secureComms=

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
Updated the relevant files for:

Set mutual TLS solr.secureComms=https and port.ssl

The work for this started with #312 

Cheers

### CODE REVIEW

File: insight-engine/latest/install/options.md [hunk 1]
```diff
@@ -203,9 +203,9 @@ This task assumes you have:
 
     ```text
     index.subsystem.name=solr6
-    solr.secureComms=secret
-    solr.port=8983
-    solr.sharedSecret=password
+    mutual-TLS-solr.secureComms=https
```
Comment (tpage-alfresco) [inline insight-engine/latest/install/options.md:206]: I can't find any information about this property name being changed. Where are you getting this from?

Comment (oliveralfresco2018) [reply insight-engine/latest/install/options.md:206]: @tpage-alfresco  it looks like it has come from this PR https://github.com/Alfresco/docs-alfresco/pull/312
If you dont agree with the PR I am happy to close it !

Comment (tpage-alfresco) [reply insight-engine/latest/install/options.md:206]: I don't understand your reply. That PR doesn't mention a property called `mutual-TLS-solr.secureComms`.

Comment (oliveralfresco2018) [reply insight-engine/latest/install/options.md]: I have made this the same as the others.

File: search-services/latest/install/options.md [hunk 2]
```diff
@@ -201,9 +201,9 @@ This task assumes you have:
 
     ```text
     index.subsystem.name=solr6
-    solr.secureComms=secret
-    solr.port=8983
-    solr.sharedSecret=password
+    mutual-TLS-solr.secureComms=https
```
Comment (tpage-alfresco) [inline search-services/latest/install/options.md:204]: I think this is still wrong.

Comment (oliveralfresco2018) [reply search-services/latest/install/options.md:204]: Right Ive removed the mutual-TLS-solr.secureComms=https

File: insight-engine/latest/install/options.md [hunk 3]
```diff
@@ -203,9 +203,8 @@ This task assumes you have:
 
     ```text
     index.subsystem.name=solr6
-    solr.secureComms=secret
-    solr.port=8983
-    solr.sharedSecret=password
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply insight-engine/latest/install/options.md:208]: Right added in the secret word bit.

File: search-services/latest/install/options.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-services/latest/install/options.md:204]: @tpage-alfresco can you let me know the correct parameters here please?
im not sure what they are.
cheers.

File: insight-engine/latest/install/options.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply insight-engine/latest/install/options.md:208]: @tpage-alfresco thanks for your help.
If what I have done is not correct can you let me know what is correct please?
cheers

**Review by tpage-alfresco — APPROVED** [review]
LGTM

**Review by Amy-Hyland — APPROVED** [review]
My only feedback: In insight-engine/1.4/install/options.md (line 310) and insight-engine/latest/install/options.md (line 207) there are extra spaces after password.

**Pull request merged**